main.go

```diff
@@ -210,13 +215,15 @@ func filterPRsForTouch(client *github.Client, unfiltered []*github.PullRequest,
 	return prs
 }
 
-func prCommentedOnBy(client *github.Client, pr *github.PullRequest, users []string) bool {
+func prCommentedOnBy(client *github.Client, pr *github.PullRequest, users []string, startTime time.Time) bool {
 	page := 0
 	for {
 		c, r, err := retryListCommentsUpTo(retryCount, func() ([]*github.IssueComment, *github.Response, error) {
 			return client.Issues.ListComments(context.TODO(), pr.GetBase().GetRepo().GetOwner().GetLogin(), pr.GetBase().GetRepo().GetName(), pr.GetNumber(), &github.IssueListCommentsOptions{
+				Since: &startTime,
```

Comment:
This changes the output in a meaningful way. 

In the old code, any older comments on a PR that is eventually is merged counts. In this new code, only new comments will count. 

Maybe this implies that we should only count merged lines? At which point I would _not_ want this filter here.